tests/test_files/html/placeholders/action_links_placeholders_tabs.html

```diff
@@ -0,0 +1,4 @@
+<p><a href="https://www.youtube.com/watch?v=dQw4w%09%09%099WgXcQ"><img alt="call to action img" src="https://dev-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/vanotify-action-link.png" class="action_link"><b>click this</b></a></p>
+<p><a href="https://www.example.com/watch?t=abc%09def"><img alt="call to action img" src="https://dev-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/vanotify-action-link.png" class="action_link"><b>action link</b></a></p>
+<p><a href="https://www.example.com/watch?t=abc%09def"><img alt="call to action img" src="https://dev-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/vanotify-action-link.png" class="action_link"><b>click this</b></a></p>
+
```

Comment:
My understanding of the way these test files were generated was that each .md had associated html and text files. Where is the action link tab markdown and associated test?